ehitchhiking-ui/src/app/shared/components/blacklist/blacklist.component.sass

```diff
@@ -0,0 +1,36 @@
+@import "../../../../styles/colour-constants"
+@import "../../../../styles/fonts-constants"
+
+.truncated
+  white-space: nowrap
+  overflow: hidden
+  text-overflow: ellipsis
+.blacklist-title
+  position: relative
+  left: 40%
+.blacklist
+  height: 20rem
+  overflow-y: scroll
+  overflow-x: hidden
+  padding: 0rem
+  margin: 0.5rem
+  .blacklist-user
+    border: 1px solid $black
+    display: flex
+    justify-content: space-around
+    align-items: center
+    .profile-photo
+      font-size: $display-2
+    .remove-icon
+      font-size: 2rem
+      display: table
+      vertical-align: middle
+      color: $dark-red
+    .blacklist-user-name
+      font-size: 1rem
+      margin-top: 1rem
+      width: 10rem
+      @extend .truncated
+.blacklist-back-btn
+  left: 40%
```

Comment:
Magic offset. Use flex to centralize container

Reply:
does not work

Reply:
done